firestore/integration_test_internal/CMakeLists.txt

```diff
@@ -204,6 +204,7 @@ include_directories(src)
 include_directories(${FIREBASE_CPP_SDK_DIR})
 # Additional public headers from Firestore core SDK.
 include_directories(${PROJECT_BINARY_DIR}/bin/external/src/firestore)
+include_directories(${PROJECT_BINARY_DIR}/bin/external/src/nanopb)
```

Comment:
I think you also need to add the following line:
include_directories(${PROJECT_BINARY_DIR}/bin/external/src/firestore/Firestore/Protos/nanopb)